annotation-file-utilities/src/annotator/find/Insertion.java

```diff
@@ -203,6 +247,50 @@ public static void setAlwaysQualify(Set<String> set) {
     }
 
     /**
+     * Gets the type.  It is assumed that the returned value will be
+     * modified to update the type to be inserted.
+     */
+    public Type getType() {
+        return type;
+    }
+
+    /**
+     * Sets the type.
+     */
+    public void setType(Type type) {
+        this.type = type;
+    }
+
+    public DeclaredType getBaseType() {
+        return getBaseType(type);
+    }
+
+    public static DeclaredType getBaseType(Type type) {
+        if (type != null) {
+            switch (type.getKind()) {
+            case DECLARED:
+                return (DeclaredType) type;
+            case BOUNDED:
+                return getBaseType(((BoundedType) type).getType());
+            case ARRAY:
+                return getBaseType(((ArrayType) type).getComponentType());
+            }
+        }
+        return null;
+    }
+
+    /**
+     * Indicates whether inserted text should consist <em>only</em> of
+     *  the indicated annotations.  Useful when subordinate insertions
```

Comment:
Can you clarify "subordinate insertions", or give an example or two of a subordinate insertion?